### PR TITLE
Fixed installing config files from proper dir based on product version

### PR DESCRIPTION
**Pull request by webhdx**
This PR fixes detecting correct directory based on product version. It should handle all cases:
- stable version: ie. `3.3.0` will use directories: `3.3`, `3.3.0`
- branch: i.e.`dev-my_branch` will look for directory `dev-my_branch`
- branch using alias: i.e. `dev-master` will first look for dir named after alias `3.3.x-dev` and then `dev-master`

### CODE REVIEW

File: resources/platformsh/ibexa-commerce/3.3/.platform/configsets/solr6/conf/solrconfig.xml.bak [hunk 1]
```diff
@@ -0,0 +1,1387 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (alongosz) [inline resources/platformsh/ibexa-commerce/3.3/.platform/configsets/solr6/conf/solrconfig.xml.bak:1]: This garbage can be removed. This is a left-over from sed operation in `install-solr.sh` which shouldn't be committed in a first place. It's already in a proper form in `resources/platformsh/ibexa-commerce/3.3/.platform/configsets/solr6/conf/solrconfig.xml`

File: resources/platformsh/ibexa-commerce/3.3/.platform/configsets/solr6/conf/solrconfig.xml.bak2 [hunk 2]
```diff
@@ -0,0 +1,1330 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (alongosz) [inline resources/platformsh/ibexa-commerce/3.3/.platform/configsets/solr6/conf/solrconfig.xml.bak2:1]: same here

**Review by alongosz — CHANGES_REQUESTED** [review]
`.platform/configsets/solr6/conf/solrconfig.xml.bak` and .bak2 garbage can be removed (there are 2 more which I cannot mark).

**Review by alongosz — APPROVED** [review]

**Review by mnocon — APPROVED** [review]
There is one issue - https://issues.ibexa.co/browse/EZP-32292 but I've discussed it with Maciek and it's not blocking us right now

**Pull request merged**